frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/edit-fieldtype/EditDropdown/EditDropdown.tsx

```diff
@@ -72,6 +76,14 @@ export const EditDropdown = ({ field }: EditDropdownProps): JSX.Element => {
     [],
   )
 
+  const isFieldUsedForConditionalRouting = useMemo(() => {
```

Comment:
Added since should only display the inline message if the dropdown field being edited is used in the workflow builder for cond routing